roles/openshift_metrics/files/templates/cassandra_serviceaccount.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
I see the distinction here.  You may have answered this in a following comment, but maybe we should consider a naming convention instead of dropping them in a sub-folder if we wish to keep a static openshift template. Something like:  files/cassandra_serviceaccount_template.yaml